JHUGenLexicon/src/JHUGenLexiconTranslator.cc

```diff
@@ -484,41 +722,75 @@ std::vector<std::vector<double>> JHUGenLexiconTranslator::getTranslationMatrix(
         res[coupl_efthbasis_tCza][coupl_eftjhu_ghzgs4] = -((2*sqrt(cw)*sqrt(sw))/pow(e,2));
         res[coupl_efthbasis_Caa][coupl_eftjhu_ghgsgs2] = -(2/pow(e,2));
         res[coupl_efthbasis_tCaa][coupl_eftjhu_ghgsgs4] = -(2/pow(e,2));
-        res[coupl_efthbasis_Cgg][coupl_eftjhu_ghg2] = -2/1.0;
+        res[coupl_efthbasis_Cgg][coupl_eftjhu_ghg2] = -2.0/1.0;
```

Comment:
I was referring to this line vs the others. My assumption is that 2.0 vs 2 does not make a change since /1.0 already casts 2 to double, is that correct?

Reply:
Yes this is correct. This shouldn't change the type of this entry of the matrix at all.

Reply:
and it didnt